ods_ci/tasks/Tasks/rhods_olm.robot

```diff
@@ -15,7 +15,11 @@ ${RHODS_OSD_INSTALL_REPO}    None
 ${TEST_ENV}                  AWS
 ${INSTALL_TYPE}              OperatorHub
 ${UPDATE_CHANNEL}            odh-nightlies
+${MANAGE_CODEFLARE_OPERATOR} False
```

Comment:
need one more space before the value. 

You can set it to `${FALSE}` so you can avoid `Convert to boolean` later

Reply:
Nice! Just checking, will it work when I set the value in the Jenkins job as follows?
```
--test-variable MANAGE_CODEFLARE_OPERATOR:false
```
or
```
--test-variable MANAGE_CODEFLARE_OPERATOR:true
```
I mean, if the value is set as the above string, will it be converted into a boolean automatically?

Reply:
the RF UserGuide says:
`Strings TRUE, YES, ON and 1 are converted to True,` so probably if we pass "TRUE" instead of "true" it should work.

Probably, to be on safer side, we could use Convert to Boolean as you did.

Reply:
Updated, will test it.

Reply:
I have found that Keywords cannot be called from the Variables definition as I did: https://groups.google.com/g/robotframework-users/c/salOiBEzPXE

I have introduced a new keyword `Is CodeFlare Managed` and will use that instead, testing it..

Reply:
Looks good so far.